k256/src/arithmetic/scalar.rs

```diff
@@ -20,6 +20,14 @@ const MODULUS: [u64; LIMBS] = [
     0xFFFF_FFFF_FFFF_FFFF,
 ];
 
+/// Constant representing the modulus / 2
+const FRAC_MODULUS_2: [u64; LIMBS] = [
```

Comment:
I borrowed the `FRAC_*_N` naming from [core::f64::consts](https://doc.rust-lang.org/core/f64/consts/index.html). Not sure if there's a better convention I should be using?